sdcm/utils/nemesis_jobs_configs.py

```diff
@@ -0,0 +1,38 @@
+# pylint: disable=line-too-long
+NEMESIS_ADDITIONAL_CONFIGS = {
+    "PublicExtraInterfaceRequired": {
```

Comment:
won't it be better to save those a yamls ? and not generate multiple of those ?

Reply:
Moved to yamls, removed one level of indirection